tests/test_package/test_convert.py

```diff
@@ -0,0 +1,16 @@
+from tidy3d.web.cli import tidy3d_cli
+from tidy3d.web.cli.converter import converter_arg
+from pathlib import Path
+
+
+def test_tidy3d_converter():
+    lsf_path = str(Path.cwd()) + "/tests/data/example.lsf"
+    converter_arg(lsf_path, "lsf_converter_test.py")
+    file_path = Path("lsf_converter_test.py")
+    assert file_path.exists()
+    Path.unlink(file_path)
+    lsf_path = str(Path.cwd()) + "/tests/data/monitors.lsf"
```

Comment:
the 2nd half is duplicating the first. There's a pytest tool called [parameterize](https://docs.pytest.org/en/7.3.x/how-to/parametrize.html) that we use often to test on arrays of inputs, let's implement that here. You can see an example [here](https://github.com/flexcompute/tidy3d/blob/develop/tests/test_components/test_custom.py#L77-L82). 

The idea is to simply pass the input file name to the test function and then use the same logic to test each one.

```py
@pytest.mark.parameterize(...) # set filename values here
def test_tidy3d_converter(filename):
    # construct path out of supplied filename
```